apps/src/codebridge/Console/console.module.scss

```diff
@@ -1,4 +1,5 @@
 @import 'color.scss';
```

Comment:
Could you explain why we're not using dsco color variables for some of these colors in this stylesheet? Looks like the other style sheets are only using dsco colors.

Reply:
Gotcha - thanks for the explanation!